python/paddle/fluid/contrib/slim/quantization/quantization_mkldnn_pass.py

```diff
@@ -534,13 +535,12 @@ def _find_avg_pooling_ids(self, graph):
 
     def _quantize_fp32_graph(self, graph):
         ir_pass = self._core.get_pass('cpu_quantize_placement_pass')
-        inference_program = graph.to_program()
-        ir_graph = self._core.Graph(inference_program.desc)
+        cpp_graph = graph.graph
         ir_pass.set('quantize_enabled_op_types', {'conv2d', 'pool2d'})
         ir_pass.set('quantize_excluded_op_ids',
                     self._find_avg_pooling_ids(graph))
-        ir_pass.apply(ir_graph)
-        graph = IrGraph(ir_graph, for_test=True)
+        ir_pass.apply(cpp_graph)
+        graph = IrGraph(cpp_graph, for_test=True)
```

Comment:
`graph = IrGraph(cpp_graph, for_test=True)` can be removed. Have a try please.

Reply:
Same as above.